src/frontends/pytorch/src/op_table.cpp

```diff
@@ -425,6 +425,7 @@ const std::unordered_map<std::string, CreatorFunction> get_supported_ops_ts() {
         {"aten::col2im", op::translate_col2im},
         // aten::complex - Supported in limited set of patterns
         {"aten::concat", op::translate_cat},
+        {"aten::concatenate", op::translate_cat},
```

Comment:
how to reproduce such case in layer tests?

Reply:
this is rare case, when something went wrong with normalization in tracing, I think this is difficult to catch such behaviour in layer test and we can trust to torch aliasing system that torch.concatenate is alias for torch.cat
https://github.com/pytorch/pytorch/blob/dbed747aae223d53ca4e22fe45c24d1d9a8b4432/torch/csrc/jit/passes/normalize_ops.cpp#L131